bot/cogs/defcon.py

```diff
@@ -104,6 +104,7 @@ def mod_log(self) -> ModLog:
                     log.exception(f"Unable to send rejection message to user: {member}")
 
                 await member.kick(reason="DEFCON active, user is too new")
+                self.bot.stats.incr("defcon_leaves")
```

Comment:
Naming this `defcon.leaves` would be more consistent with the naming scheme used elsewhere.